src/IO/S3Common.cpp

```diff
@@ -631,7 +633,7 @@ namespace S3
             /// https://docs.aws.amazon.com/awscloudtrail/latest/userguide/cloudtrail-s3-bucket-naming-requirements.html
             if (bucket.length() < 3 || bucket.length() > 63)
                 throw Exception(
```

Comment:
Let's also change to fmt-style Exception constructor.

Reply:
+